docs/ug/runSh.md

```diff
@@ -28,4 +28,5 @@ Depending on which version you wish to use for report generation, add one of the
 * `--release`: Use the latest release (Stable)
 * `--master`: Use the latest version of the master branch
 * `--tag TAG` (e.g. `--tag v1.6.1`): Use the version identified by the Git tag given
+* `--latest TAG_PREFIX`: Gets the latest version with the given tag prefix (e.g. `--latest v1.6` can get `v1.6.1`)
```

Comment:
I've modified the phrasing to match the rest of the flags.
```suggestion
* `--latest TAG_PREFIX`: Use the latest version with the given tag prefix (e.g. `--latest v1.6` can use `v1.6.1`)
```

Reply:
Committed